src/App.js

```diff
@@ -1,19 +1,48 @@
 import React from "react";
-import Container from "react-bootstrap/Container";
 import "./App.css";
 import Header from "./Header.js";
 import Main from "./Main.js";
 import Footer from  "./Footer.js";
-import beasImgs from  "./data.json";
+import beastImgs from  "./data.json";
+import SelectedBeast from "./SelectedBeast";
+
 
 class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      show: false,
+      selectedB:{}
```

Comment:
As far as naming conventions go, this is a little hard to understand is all. If I didn't already know what this application did I'd really have to read through your program to understand what this bit of state was all about.